src/epub/content.opf

```diff
@@ -53,7 +53,7 @@
 		<meta id="long-description" property="se:long-description" refines="#description">
 			&lt;p&gt;&lt;i&gt;The Prisoner of Zenda&lt;/i&gt; by &lt;a href="https://standardebooks.org/ebooks/anthony-hope"&gt;Anthony Hope&lt;/a&gt; is an adventure novel first published in 1894 that takes place in the fictional Kingdom of Ruritania.&lt;/p&gt;
 			&lt;p&gt;It tells the story of Rudolph Rassendyll, who is, because of past indiscretions in the family and unbeknownst to him, the near twin of King Rudolph V of Ruritania. Labeled a ne’er-do-well by his sister-in-law, young Rudolph determines to escape his family and secretly travel to Ruritania for the coronation of his distant relative. But when the king is drugged and abducted on the eve of this ceremony, young Rassendyll is convinced to take his place to try and save the day.&lt;/p&gt;
-			&lt;p&gt;But things don’t go as planned as the conspirators fail to reckon with the king’s brother, the dastardly Duke of Strelsau or his fiancée, the beautiful Princess Flavia. What follows is a tale of bravery, sacrifice and love, filled with romance and feats of derring-do that still stands the test of time.&lt;/p&gt;
+			&lt;p&gt;But things don’t go as planned as the conspirators fail to reckon with the king’s brother, the dastardly Duke of Strelsau; or his fiancée, the beautiful Princess Flavia. What follows is a tale of bravery, sacrifice and love, filled with romance and feats of derring-do that still stands the test of time.&lt;/p&gt;
```

Comment:
I think we should leave this as a comma and not a semicolon

Reply:
Switched to comma.